### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/deployment/docker.md`

### PR DESCRIPTION
**Pull request by xzmeng**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2626f65623dfaf48b6520acfba6faaa7119335c7 at: https://60549044.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/docker.md [hunk 1]
```diff
@@ -0,0 +1,728 @@
+# 容器中的 FastAPI - Docker
+
+部署 FastAPI 应用程序时，常见的方法是构建 **Linux 容器镜像**。 通常使用 <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> 完成。 然后，您可以通过几种可能的方式之一部署该容器映像。
+
+使用 Linux 容器有几个优点，包括**安全性**、**可复制性**、**简单性**等。
+
+!!! tip
+    很着急并且已经知道这些东西了？ 跳转到下面的 [`Dockerfile` 👇](#build-a-docker-image-for-fastapi)。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:8]: 链接无法跳转

Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:8]:
```suggestion
    赶时间并且已经知道这些东西了？ 跳转到下面的 [`Dockerfile` 👇](#为-fastapi-构建-docker-镜像)。
```

File: docs/zh/docs/deployment/docker.md [hunk 2]
```diff
@@ -0,0 +1,728 @@
...
+
+# If running behind a proxy like Nginx or Traefik add --proxy-headers
+# CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80", "--proxy-headers"]
+```
+
+</details>
+
+## 什么是容器
+
+容器（主要是 Linux 容器）是一种非常**轻量级**的打包应用程序的方式，包括其所有依赖项和必要的文件，同时使它们与同一系统中的其他容器（其他应用程序或组件）隔离。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:35]:
```suggestion
容器（主要是 Linux 容器）是一种非常**轻量级**的打包应用程序的方式，其包括所有依赖项和必要的文件，同时它们可以和同一系统中的其他容器（或者其他应用程序/组件）相互隔离。
```

File: docs/zh/docs/deployment/docker.md [hunk 3]
```diff
@@ -0,0 +1,728 @@
...
+
+Linux 容器使用主机（机器、虚拟机、云服务器等）的相同 Linux 内核运行。 这仅仅意味着它们非常轻量（与模拟整个操作系统的完整虚拟机相比）。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:37]:
```suggestion
Linux 容器使用宿主机（如物理服务器、虚拟机、云服务器等）的Linux 内核运行。 这意味着它们非常轻量（与模拟整个操作系统的完整虚拟机相比）。
```

File: docs/zh/docs/deployment/docker.md [hunk 4]
```diff
@@ -0,0 +1,728 @@
...
+
+这样，容器消耗**很少的资源**，与直接运行进程相当（虚拟机会消耗更多）。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:39]:
```suggestion
通过这样的方式，容器消耗**很少的资源**，与直接运行进程相当（虚拟机会消耗更多）。
```

File: docs/zh/docs/deployment/docker.md [hunk 5]
```diff
@@ -0,0 +1,728 @@
...
+
+容器还有自己的**隔离**运行进程（通常只有一个进程）、文件系统和网络，简化了部署、安全、开发等。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:41]:
```suggestion
容器的进程（通常只有一个）、文件系统和网络都运行在隔离的环境，这简化了部署、安全、开发等。
```

File: docs/zh/docs/deployment/docker.md [hunk 6]
```diff
@@ -0,0 +1,728 @@
...
+
+## 什么是容器镜像
+
+**容器**是从**容器镜像**运行的。
+
+容器镜像是容器中应存在的所有文件、环境变量和默认命令/程序的**静态**版本。 **静态**这里的意思是容器**镜像**没有运行，没有被执行，只是打包的文件和元数据。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:47]:
```suggestion
容器镜像是容器中文件、环境变量和默认命令/程序的**静态**版本。 **静态**这里的意思是容器**镜像**还没有运行，只是打包的文件和元数据。
```

File: docs/zh/docs/deployment/docker.md [hunk 7]
```diff
@@ -0,0 +1,728 @@
...
+
+与存储静态内容的“**容器镜像**”相反，“**容器**”通常指正在运行的实例，即正在**执行的**。
+
+当**容器**启动并运行时（从**容器镜像**启动），它可以创建或更改文件、环境变量等。这些更改将仅存在于该容器中，但不会持久存在于底层中 容器镜像（不会保存到磁盘）。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:51]:
```suggestion
当**容器**启动并运行时（从**容器镜像**启动），它可以创建或更改文件、环境变量等。这些更改将仅存在于该容器中，而不会持久化到底层的容器镜像中（不会保存到磁盘）。
```

File: docs/zh/docs/deployment/docker.md [hunk 8]
```diff
@@ -0,0 +1,728 @@
...
+
+容器镜像相当于**程序**文件和内容，例如 `python` 和某些文件 `main.py`。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:53]:
```suggestion
容器镜像相当于**程序**和文件，例如 `python`命令 和某些文件 如`main.py`。
```

File: docs/zh/docs/deployment/docker.md [hunk 9]
```diff
@@ -0,0 +1,728 @@
...
+
+# (9)
+RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+
+# (10)
+COPY ./app /code/app
+
+# (11)
+CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80"]
+```
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:659]: 没有正确渲染

File: docs/zh/docs/deployment/docker.md [hunk 10]
```diff
@@ -0,0 +1,728 @@
...
+COPY ./requirements.txt /code/requirements.txt
+
+RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+
+# (1)
+COPY ./main.py /code/
+
+# (2)
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "80"]
+```
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:380]: 没有正确渲染

File: docs/zh/docs/deployment/docker.md [hunk 11]
```diff
@@ -0,0 +1,728 @@
...
+     在开发过程中一次又一次构建映像时，在此步骤中使用缓存将为您节省大量**时间**，而不是**每次**都**下载和安装**所有依赖项。
+
+
+5. 将“./app”目录复制到“/code”目录中。
+
+     由于其中包含**更改最频繁**的所有代码，因此 Docker **缓存**不会轻易用于此操作或任何**后续步骤**。
+
+     因此，将其放在`Dockerfile`**接近最后**的位置非常重要，以优化容器映像的构建时间。
+
+6. 设置**命令**来运行 `uvicorn` 服务器。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:227]: 渲染错位

**Review by socket-socket — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6eccce5227b94475cd2431c0d7684f33effe5d9f at: https://908f6858.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/docker.md [hunk 12]
```diff
@@ -0,0 +1,728 @@
...
+
+5. 生成`requirements.txt`文件。
+
+6. 这是最后阶段，这里的任何内容都将保留在最终的容器映像中。
+
+7. 将当前工作目录设置为`/code`。
+
+8. 将 `requirements.txt` 文件复制到 `/code` 目录。
+
+     该文件仅存在于前一个阶段，这就是为什么我们使用 `--from-requirements-stage` 来复制它。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/docker.md:681]:
```suggestion
   该文件仅存在于前一个阶段，这就是为什么我们使用 `--from-requirements-stage` 来复制它。
```

**Review by socket-socket — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 953ab49fd8657885ebe92c98836d3a562ac1ddbb at: https://ae37bcd7.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/docker.md [hunk 13]
```diff
@@ -0,0 +1,728 @@
...
+
+2. 将 `/tmp` 设置为当前工作目录。
+
+     这是我们生成文件`requirements.txt`的地方
+
+3. 在此阶段安装Poetry。
+
+4. 将`pyproject.toml`和`poetry.lock`文件复制到`/tmp`目录。
+
+   因为它使用 `./poetry.lock*` （以 `*` 结尾），所以如果该文件尚不可用，它不会崩溃。
```
Comment (xzmeng) [inline docs/zh/docs/deployment/docker.md:671]:
```suggestion
     因为它使用 `./poetry.lock*` （以 `*` 结尾），所以如果该文件尚不可用，它不会崩溃。
```

File: docs/zh/docs/deployment/docker.md [hunk 14]
```diff
@@ -0,0 +1,728 @@
...
+
+5. 生成`requirements.txt`文件。
+
+6. 这是最后阶段，这里的任何内容都将保留在最终的容器映像中。
+
+7. 将当前工作目录设置为`/code`。
+
+8. 将 `requirements.txt` 文件复制到 `/code` 目录。
+
+   该文件仅存在于前一个阶段，这就是为什么我们使用 `--from-requirements-stage` 来复制它。
```
Comment (xzmeng) [inline docs/zh/docs/deployment/docker.md:681]:
```suggestion
     该文件仅存在于前一个阶段，这就是为什么我们使用 `--from-requirements-stage` 来复制它。
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 52a2f7f52d090869e21bd0c54f4787bb7de5ad94 at: https://426906b0.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 053ce6139f3473065cc33c6e71fd9157730f4480 at: https://4b99b2c6.fastapitiangolo.pages.dev

**Review by Aruelius — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for your work! @xzmeng 🤓 

And thanks for the reviews @Aruelius and @socket-socket 🚀💯

**Pull request merged**